src/rsdiv/evaluation/diversity_metrics.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+
+def gini_coefficient(x: np.ndarray, sort: bool = True) -> float:
+    if sort:
+        x = np.sort(x)[::-1]
+    count: int = x.shape[0]
+    b: float = x @ np.arange(1, count + 1)
```

Comment:
avoid to use the single character as the variable's name. Try to replace x, b as more less confusing names for users.

Reply:
a, b here correspond to areas in [gini's definition](https://en.wikipedia.org/wiki/File:Economics_Gini_coefficient2.svg). I will try to make them more understandable.